src/config.rs

```diff
@@ -827,6 +843,11 @@ impl Context {
                     .await?;
                 }
             }
+            Config::StatsSending => {
+                self.sql.set_raw_config(key.as_ref(), value).await?;
```

Comment:
Maybe do this as the last step so as if `StatsSending` is set, `LastCountedMsgId` and `LastOldContactId` are guaranteed to be correct, even if the program crashes in between?

Also if `StatsSending` is already set, this updates `LastCountedMsgId`, but it mustn't.